config/compiler.intel.mk

```diff
@@ -14,7 +14,7 @@ FFLAGS += -qopenmp
 F90FLAGS += -qopenmp
 LFLAGS += -qopenmp
 endif
-CTM_FLAGS = $(LFLAGS)
+CTM_LFLAGS = $(LFLAGS)
```

Comment:
This change is just fixing a minor oversight introduced in #62, as discussed in the meeting.